Gameserver/src/main/org.l2j.gameserver/org/l2j/gameserver/model/stats/Formulas.java

```diff
@@ -728,7 +729,24 @@ public static double calculateSkillResurrectRestorePercent(double baseRestorePer
         return restorePercent;
     }
 
+    public static boolean calcPhysicalFailure(Creature activeChar, Creature target, Skill skill ) {
+        if (target instanceof Player || target instanceof Summon)
+            return false;
+        if ((target.getLevel() - activeChar.getActingPlayer().getLevel()) >= 11 ) {
```

Comment:
Miss every attack if the target is 11 level above attacker seems not right.

![Shot00006](https://user-images.githubusercontent.com/5034192/110854961-77b42f00-8294-11eb-91b6-3d689da0ea34.png)